test/src/engine/BMPlayerTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * Generated by PHPUnit_SkeletonGenerator 1.2.0 on 2012-12-01 at 14:50:59.
+ */
+class BMPlayerTest extends PHPUnit_Framework_TestCase {
+
+    /**
+     * @var BMPlayer
+     */
+    protected $object;
+
+    /**
+     * Sets up the fixture, for example, opens a network connection.
+     * This method is called before a test is executed.
+     */
+    protected function setUp() {
+        $this->object = new BMPlayer;
+    }
+
+    /**
+     * Tears down the fixture, for example, closes a network connection.
+     * This method is called after a test is executed.
+     */
+    protected function tearDown() {
+
+    }
+
+    public function test_dummp() {
```

Comment:
You should probably go so far as to call this `test_dummy`...

But also don't you need some real tests of your new class?